VSEmbed.DemoApp/VSEmbed.DemoApp.csproj

```diff
@@ -41,6 +41,18 @@
       <HintPath>..\packages\BasicUndo.0.9.3\lib\net45\BasicUndo.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.VisualStudio.Editor.Implementation, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
```

Comment:
We usually remove specificversion=false, is it needed here?
It look like VS is adding this tag automatically